lib/components/Upload.jsx

```diff
@@ -158,7 +158,7 @@ var Upload = React.createClass({
     return (
       <div className="Upload-button">
         <button
-          className="btn btn-secondary"
+          className="btn btn-primary"
```

Comment:
Ah, my bad. I didn't realize the button style that was in the version I branched off of wasn't to date with Sara's design.